docker/package/tezos_setup_wizard.py

```diff
@@ -222,6 +222,37 @@ def check_blockchain_data(self):
             return False
         return True
 
+    # Check <network>.xtz-shots.io/base.json and collect the most recent snapshot
+    # that is suited for the chosen history mode
+    def get_snapshot_link(self):
+        self.config["snapshot_url"] = None
+        network_url = f"https://{self.config['network']}.xtz-shots.io"
+        json_url = f"{network_url}/base.json"
+        try:
+            with urllib.request.urlopen(json_url) as url:
+                snapshot_array = json.load(url)
+        except (urllib.error.URLError, ValueError):
+            print(f"Couldn't collect snapshot metadata from {json_url}")
+            return
+
+        for artifact in snapshot_array:
```

Comment:
The structure of metadata JSON is highly likely going to change in the future, it'll probably be an unexpected change for us and users. What do you think about using a fallback option (e.g. use the previous `https://<network>.xtz-shots.io/rolling`) in case the JSON that we got has some unexpected format? This fallback workaround will be removed once the format of JSON will be stabilized

Reply:
I'd also like to remind that we might not want to merge this right away and wait a bit to see if out feedback is taken care of.

In fact, I'd recommend setting this as a `Draft` temporarily